src/components/DappBrowser/BrowserTab.tsx

```diff
@@ -262,9 +263,14 @@ export const BrowserTab = React.memo(function BrowserTab({ tabIndex, injectedJS
   const [backgroundColor, setBackgroundColor] = useState<string>();
 
   const createMessengers = useCallback((origin: string, tabId: string) => {
-    if (!webViewRef.current) return;
+    console.log('creating messengers');
```

Comment:
boooooo

Reply:
be careful bro https://github.com/search?q=repo%3Arainbow-me%2Frainbow+echo+%27hi&type=code

Reply:
lmao